zanata-model/src/main/java/org/zanata/model/HTextFlowTarget.java

```diff
@@ -117,10 +118,25 @@
 
     private List<HTextFlowTargetReviewComment> reviewComments;
 
+    @Getter
+    private String revisionComment;
+
+    private boolean revisionCommentChanged = false;
+
     // Only for internal use (persistence transient)
     @Setter(AccessLevel.PRIVATE)
     private Integer oldVersionNum;
 
+    public void setRevisionComment(String revisionComment) {
+        this.revisionComment = revisionComment;
+        revisionCommentChanged = true;
```

Comment:
Maybe it is the same. Suggest wrapping both lines in a null-safe check that the old and new value are different.